lib/c/criu.h

```diff
@@ -157,6 +157,35 @@ int criu_restore_child(void);
 typedef void *criu_predump_info;
 int criu_dump_iters(int (*more)(criu_predump_info pi));
 
+/*
+ * Get the version of the actual binary used for RPC.
+ *
+ * As this library is just forwarding all tasks to an
+ * independent (of this library) CRIU binary, the actual
+ * version of the CRIU binary can be different then the
+ * hardcoded values in the libary (version.h).
+ * To be able to easily check the version of the CRIU binary
+ * the function criu_get_version() returns the version
+ * in the following format:
+ *
+ * (major * 10000) + (minor * 100) + sublevel
```

Comment:
Maybe it is better to use bit shift based version format? If for instance 128 is enough for version number we can have:
```
#define V_MAJOR_SHIFT 14
#define V_MINOR_SHIFT 7
(major<<V_MAJOR_SHIFT) + (minor<<V_MINOR_SHIFT) + sublevel
```

Reply:
If I understand it correctly that would be pretty difficult to read (at least for me).

Instead of getting 31400 for CRIU 3.14 I would get 50944 ((3 << 14)+(14 << 7)). Especially for the check_version() function it would not be very obvious in the code for which version I want to check. Not convinced that this would make sense.

Reply:
May be I chouse bad shift, 8 looks nice:

>>> hex(((3 << 16)+(14 << 8)))
'0x30e00'

Reply:
I would prefer to leave it the way it is to easily read number when in decimal.

runc uses the decimal approach already for CRIU version detection and the go-criu code (which is a copy of the runc code) also